src/rpc/claimtrie.cpp

```diff
@@ -387,9 +399,20 @@ UniValue getclaimbyid(const UniValue& params, bool fHelp)
         claim.push_back(Pair("txid", claimValue.outPoint.hash.GetHex()));
         claim.push_back(Pair("n", (int) claimValue.outPoint.n));
         claim.push_back(Pair("amount", claimValue.nAmount));
-        claim.push_back(Pair("effective amount",
-                             pclaimTrie->getEffectiveAmountForClaim(name, claimValue.claimId)));
+        claim.push_back(Pair("effective amount", effectiveAmount));
+        UniValue supportList(UniValue::VARR);
+        BOOST_FOREACH(const CSupportValue& support, supports) {
+            UniValue supportEntry(UniValue::VOBJ);
+            supportEntry.push_back(Pair("txid", support.outPoint.hash.GetHex()));
+            supportEntry.push_back(Pair("n", (int)support.outPoint.n));
+            supportEntry.push_back(Pair("height", support.nHeight));
+            supportEntry.push_back(Pair("valid at height", support.nValidAtHeight));
+            supportEntry.push_back(Pair("amount", support.nAmount));
+            supportList.push_back(supportEntry);
+        }
+        claim.push_back(Pair("supports", supportList));
         claim.push_back(Pair("height", claimValue.nHeight));
+        claim.push_back(Pair("valid at height", claimValue.nValidAtHeight));
```

Comment:
"valid at height" field was added here but not to the doc string

Reply:
@kaykurokawa Which docstring? It was added to the help string.

Reply:
its added when listing fields for supports, but not to the claim itself in the docstring

Reply:
Ah, I see, thanks!

Reply:
Updated